spyder/plugins/__init__.py

```diff
@@ -37,6 +37,7 @@
 from spyder.py3compat import configparser, is_text_string
 from spyder.utils import icon_manager as ima
 from spyder.utils.qthelpers import create_action, toggle_actions
+from spyder.plugins.configdialog import PluginConfigPage  # Plugin compatibility

```

Comment:
Well, yes, this also makes the code cleaner because now all Config classes are placed in `spyder.plugins.configdialog`. So please remove it because of that :-)